src/components/Main.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+import { POD } from "./POD";
+const API_KEY = "aLF9OyI9eUKScjU0GIazR3BKJqntNfay4NXmE6lT";
+
+const style = {
+  background: "#A0BCC2",
+  height: "60vh",
+  maxWidth: "89vw",
+  margin: "1vh auto",
+  padding: "1vh",
+};
+
+const inputStyle = {
+  display: "block",
+  margin: "1rem auto",
+};
+
+export const Main = () => {
+  const [formatedDate, setFormatedDate] = useState("");
+  const [pic, setPic] = useState(null);
+  const [error, setError] = useState(false);
+  const [date, setDate] = useState(new Date().toISOString().substring(0, 10));
+  const [wrongDate, setWrongDate] = useState(null);
+  const [url, setUrl] = useState(
+    `https://api.nasa.gov/planetary/apod?api_key=${API_KEY}`
+  );
+
+  useEffect(() => {
+    if (date !== new Date().toISOString().substring(0, 10)) {
+      setUrl(
+        `https://api.nasa.gov/planetary/apod?api_key=${API_KEY}&date=${date}`
+      );
+    } else {
+      setUrl(`https://api.nasa.gov/planetary/apod?api_key=${API_KEY}`);
+    }
+  }, [date]);
+
+  useEffect(() => {
+    setPic(null);
+    fetch(url)
```

Comment:
Your app is looking good with the current changes but it's a good practice to abstract the fetching logic and states related to a custom hook so with this approach the components are more stateless and easier to apply tests.
Feel free to not implement the changes now, but it's a best practice to have custom hooks.
[Article How to fetch data with React Hooks](https://www.robinwieruch.de/react-hooks-fetch-data/)
[CodeSandbox useFetch](https://codesandbox.io/s/usereducer-example-71pbj8?file=/src/App.js)